modules/standard/LinkedLists.chpl

```diff
@@ -193,28 +193,80 @@ record LinkedList {
      if boundsChecking && size < 1 {
        HaltWrappers.boundsCheckHalt("pop_front on empty list");
      }
-     var oldfirst = first!;
-     var newfirst = first!.next;
+     var oldfirst = _first!;
+     var newfirst = _first!.next;
      var ret = oldfirst.data;
-     first = newfirst;
-     if last == oldfirst then last = newfirst;
+     _first = newfirst;
+     if _last == oldfirst then _last = newfirst;
      size -= 1;
      delete oldfirst;
      return ret;
    }
 
+  /*
+    find `e` in the list.
```

Comment:
How about "Returns `true` if this list contains an element equal to the value of `x`, and `false` otherwise."?